teams/templates/distribute_teams.html

```diff
@@ -0,0 +1,50 @@
+{% extends "base.html" %}
+{% load static %}
+{% load teams_tags %}
+{% block content %}
+
+<div class="h-100">
+    <article class="card shadow">
+        <div class="card-body">
+
+        {% if teams %}
+        {% for team_name, team_members in teams.items %}
+        <section class="card shadow">
+            <div class="card-header">{{team_name}}  <span style="float:right">Team Score: {{ team_members|calculate_team_level }}</span></div>
+            <div class="card-body">
+                <ol id="{{team_name}}" ondrop="drop(event)" ondragover="allowDrop(event)">
```

Comment:
Ah - this is where the `ondrop` is located... no `canvas` required here, I see. Ignore previous comment about canvas.

Reply:
cool